neptune_lightgbm/impl/__init__.py

```diff
@@ -79,7 +101,79 @@ def __init__(self, run: 'neptune.Run', base_namespace=''):
         self._run['source_code/integrations/neptune-lightgbm'] = __version__
 
     def __call__(self, env):
```

Comment:
It seems that you dropped some features: logging params, and logging feature names. Can you add them as well?

https://github.com/neptune-ai/neptune-client/blob/kk/lgbm/integrations/lightgbm/neptune_lightgbm/impl/__init__.py#L93:L115